examples/scenes/src/pico_svg.rs

```diff
@@ -0,0 +1,271 @@
+// Copyright 2024 the Vello Authors
+// SPDX-License-Identifier: Apache-2.0 OR MIT
+
+//! A loader for a tiny fragment of SVG
+
+use std::{num::ParseFloatError, str::FromStr};
+
+use roxmltree::{Document, Node};
+use vello::{
+    kurbo::{Affine, BezPath, Point, Size, Vec2},
+    peniko::Color,
+};
+
+pub struct PicoSvg {
+    pub items: Vec<Item>,
+    pub size: Size,
+}
+
+pub enum Item {
+    Fill(FillItem),
+    Stroke(StrokeItem),
+}
+
+pub struct StrokeItem {
+    pub width: f64,
+    pub color: Color,
+    pub path: BezPath,
+}
+
+pub struct FillItem {
+    pub color: Color,
+    pub path: BezPath,
+}
+
+struct Parser<'a> {
+    scale: f64,
+    items: &'a mut Vec<Item>,
+}
+
+impl PicoSvg {
+    pub fn load(xml_string: &str, scale: f64) -> Result<PicoSvg, Box<dyn std::error::Error>> {
```

Comment:
Why not use `anyhow::Result`? `scenes` already depends on that, and it lets us add context to the errors

Reply:
I'll give that a shot.

Reply:
It's a bit of a mess mixing the two in pico_svg, since roxml is using std:: as well.